src/woo_publications/metadata/tests/test_organisation_admin.py

```diff
@@ -0,0 +1,179 @@
+
+from woo_publications.accounts.tests.factories import UserFactory
+
+from ..constants import OrganisationOrigins
+from ..models import CUSTOM_ORGANISATION_URL_PREFIX, Organisation
+from .factories import OrganisationFactory
+
+
+@disable_admin_mfa()
+class TestOrganisationAdmin(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = UserFactory.create(
+            is_staff=True,
+            is_superuser=True,
+        )
+
+    def test_organisation_admin_show_items(self):
+        OrganisationFactory.create(
+            naam="first item",
+            oorsprong=OrganisationOrigins.municipality_list,
+            is_actief=True,
+        )
+        OrganisationFactory.create(
+            naam="second item",
+            oorsprong=OrganisationOrigins.custom_entry,
+            is_actief=True,
+        )
+        response = self.app.get(
+            reverse(
+                "admin:metadata_organisation_changelist",
+            ),
+            user=self.user,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        # test the amount of rows present
+        self.assertContains(response, "field-identifier", 2)
+
+    def test_organisation_admin_show_item_search(self):
+        organisation = OrganisationFactory.create(
+            identifier="https://www.example.com/waardenlijsten/1",
+            naam="first item",
+            oorsprong=OrganisationOrigins.municipality_list,
+            is_actief=True,
+        )
+        organisation2 = OrganisationFactory.create(
+            identifier="https://www.example.com/waardenlijsten/2",
+            naam="second item",
+            oorsprong=OrganisationOrigins.custom_entry,
+            is_actief=False,
+        )
+        url = reverse("admin:metadata_organisation_changelist")
+
+        response = self.app.get(url, user=self.user)
+
+        form = response.forms["changelist-search"]
+
+        with self.subTest("filter_on_naam"):
+            form["q"] = "first item"
+            search_response = form.submit()
+
+            self.assertEqual(search_response.status_code, 200)
+
+            # test the amount of rows present
+            self.assertContains(search_response, "field-identifier", 1)
+            self.assertContains(search_response, organisation.identifier, 1)
+
+        with self.subTest("filter_on_identifier"):
+            form["q"] = organisation2.identifier
+            search_response = form.submit()
+
+            self.assertEqual(search_response.status_code, 200)
+
+            # test the amount of rows present
+            self.assertContains(search_response, "field-identifier", 1)
+            self.assertContains(search_response, "second item", 1)
+
+        with self.subTest("filter_on_oorsprong"):
+            search_response = self.app.get(
+                url,
+                {"oorsprong__exact": OrganisationOrigins.municipality_list},
+                user=self.user,
+            )
```

Comment:
preferably use the `response.click(...)` approach here, that tests that the filters are also actually present on the page